packages/stream_chat_flutter/lib/src/message_widget.dart

```diff
@@ -830,22 +847,11 @@ class _MessageWidgetState extends State<MessageWidget>
 
   Widget get _bottomRow {
     if (isDeleted) {
-      final chatThemeData = _streamChatTheme;
-      return Row(
-        mainAxisSize: MainAxisSize.min,
-        children: [
-          StreamSvgIcon.eye(
-            color: chatThemeData.colorTheme.textLowEmphasis,
-            size: 16,
-          ),
-          const SizedBox(width: 8),
-          Text(
-            'Only visible to you',
-            style: chatThemeData.textTheme.footnote
-                .copyWith(color: chatThemeData.colorTheme.textLowEmphasis),
-          ),
-        ],
-      );
+      return widget.deletedBottomRowBuilder?.call(
+            context,
+            widget.message,
+          ) ??
+          const SizedBox.shrink();
```

Comment:
considering using an Offstage, is there any problem with that?
```suggestion
          const Offstage();
```